docs/src/guidelines.md

```diff
@@ -125,3 +125,16 @@ Note that these automerge guidelines are deliberately conservative: it is
 very possible for a perfectly good name to not pass the automatic checks and
 require manual merging. They simply exist to provide a fast path so that
 manual review is not required for every new package.
+
+## List of all GitHub PR labels that can influence AutoMerge
+
+AutoMerge reads certain labels on GitHub registration pull requests to influence it's decisions.
+Specifically, these labels are:
+
+* `Override AutoMerge: name similarity is okay`
+    * This label can be manually applied by folks with triage-level access to the registry repository.
+    * AutoMerge skips the "name similarity check" on any PRs with this label.
```

Comment:
Is it worth mentioning that this is only relevant for "new package" registrations?

Reply:
```suggestion
    * AutoMerge skips the "name similarity check" on new package registration PRs with this label.
```